domain/src/main/java/com/example/pinterbest/domain/usecases/PostLogInUseCase.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.pinterbest.domain.usecases
+
+import com.example.pinterbest.domain.entities.UserLogIn
+import com.example.pinterbest.domain.repositories.AuthRepository
+import javax.inject.Inject
+
+class PostLogInUseCase @Inject constructor(
```

Comment:
I think you can safely combine some use cases into the single one to avoid boilerplate. It's really unnecessary to create a separate use case for every action

Reply:
For more clarity, I decided to make this separation of the use-cases for now. Perhaps with further expansion of the application, the functionality will be combined